packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -2242,10 +2257,11 @@ class TextSelectionGestureDetectorBuilder {
   ///
   ///  * [TextSelectionGestureDetector.onTapDown], which triggers this callback.
   @protected
-  void onTapDown(TapDragDownDetails details) {
+  void onTapDown(TapDragDownDetails details) async {
```

Comment:
Looks like the `async` made the analyzer mad.

Could probably be fixed with an `// ignore`, or alternatively we could make the function synchronous:

```dart
      case TargetPlatform.android:
        final bool stylusEnabled;
        switch (kind) {
          case PointerDeviceKind.stylus:
          case PointerDeviceKind.invertedStylus:
            stylusEnabled = editableText.widget.stylusHandwritingEnabled;
          case PointerDeviceKind.touch:
          case PointerDeviceKind.mouse:
          case PointerDeviceKind.trackpad:
          case PointerDeviceKind.unknown:
          case null:
            stylusEnabled = false;
        }
        if (stylusEnabled) {
          Scribe.isFeatureAvailable().then((bool isAvailable) {
            if (isAvailable) {
              renderEditable.selectPosition(cause: SelectionChangedCause.tap);
              Scribe.startStylusHandwriting();
            }
          });
        }
```

Reply:
Thanks for the analyzer fixes here. I went with the synchronous idea here 👍